outreach/forms/order.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from baseframe import __
+import baseframe.forms as forms
+
+__all__ = ['LineItemForm', 'BuyerForm', 'OrderSessionForm']
+
+
+def truncate(length):
+    """
+    Return a function that truncates data to the given length.
+    To be used as part of the filters argument.
+    Eg:
+    field = forms.StringField(__("Some field"), filters=[trim(25)])
+    """
+    def _inner(data):
+        return unicode(data[:length])
+    return _inner
```

Comment:
I propose moving `truncate` into `baseframe.forms.filters` so it's generally available.